front/src/common/ReleaseInformations/ReleaseInformations.js

```diff
@@ -0,0 +1,106 @@
+// React Component displaying different applications versions
+// List of applications : Chartos, Editoast, Core, Api
+
+import React, { useEffect, useState, useContext } from 'react';
+import { useTranslation } from 'react-i18next';
+import ModalBodySNCF from 'common/BootstrapSNCF/ModalSNCF/ModalBodySNCF';
+import ModalHeaderSNCF from 'common/BootstrapSNCF/ModalSNCF/ModalHeaderSNCF';
+import { ModalContext } from 'common/BootstrapSNCF/ModalSNCF/ModalProvider';
+import { get } from '../requests';
+
+function ReleaseInformations() {
+  const { t } = useTranslation();
+  const [chartosVersion, setCurrentChartosVersion] = useState();
+  const [editoastVersion, setCurrentEditoastVersion] = useState();
+  const [coreVersion, setCurrentCoreVersion] = useState();
+  const [apiVersion, setCurrentApiVersion] = useState();
+  const { closeModal } = useContext(ModalContext);
+
+  const getChartosVersion = async () => {
+    const response = await get('/chartos/version/');
+    setCurrentChartosVersion(response.git_describe);
+  };
+
+  const getEditoastVersion = async () => {
+    const response = await get('/editoast/version/');
+    setCurrentEditoastVersion(response.git_describe);
+  };
+
+  const getCoreVersion = async () => {
+    const response = await get('/version/core/');
+    setCurrentCoreVersion(response.git_describe);
+  };
+
+  const getApiVersion = async () => {
+    const response = await get('/version/api/');
+    setCurrentApiVersion(response.git_describe);
+  };
+
+  useEffect(() => {
+    getChartosVersion();
+    getEditoastVersion();
+    getCoreVersion();
+    getApiVersion();
+  }, []);
+
+  return (
+    <>
+      <ModalHeaderSNCF className="align-items-center">
```

Comment:
Add a title explaining what is the purpose of this modal.
Why not put (not too big) OSRD logo with links to website, to the github page, like we do for this kind of functionality ?

![image](https://user-images.githubusercontent.com/6003856/214908978-4dda0788-a80e-4028-894f-91ecd377a6ad.png)
![image](https://user-images.githubusercontent.com/6003856/214909142-9b216094-07e2-4f67-9da0-64e4d8f4003d.png)